src/common/xo.js

```diff
@@ -171,6 +173,8 @@ export const subscribeGroups = createSubscription(invoke(
   sort => () => xo.call('group.getAll').then(sort)
 ))
 
+export const subscribeRoles = createSubscription(() => xo.call('role.getAll'))
```

Comment:
If that's something desirable you can sort the roles here like it was done with users or servers :)